### PR TITLE
Add GRIB2 support for Lambert conformal grids (GDT-30)

### PR DESCRIPTION
**Pull request by ajdawson**
This PR adds load support for GDT-30 (Lambert conformal) to the new GRIB2 loader.

### CODE REVIEW

**Comment by pelson** [comment]
Looks really nicely put together. Thanks @ajdawson.
I think I just need a what's new entry, and I'd be happy to merge.

**Comment by ajdawson** [comment]
I added a new commit with a what's new entry stub, I guess that is what you meant @pelson?

File: lib/iris/fileformats/grib/_load_convert.py [hunk 1]
```diff
@@ -300,6 +305,27 @@ def reference_time_coord(section):
 #
 ###############################################################################
 
+def projection_centre(projectionCentreFlag):
+    """
+    Translate the projection centre flag bitmask.
+
+    Reference GRIB2 Flag Tabel 3.5.
```
Comment (ajdawson) [inline lib/iris/fileformats/grib/_load_convert.py]: Typo: Tabel -> Table

**Comment by pelson** [comment]
:+1:

**Comment by ajdawson** [comment]
Oh dear, I just pushed the wrong branch to this... Need time to fix!

**Pull request closed without merging**